usrsctplib/CMakeLists.txt

```diff
@@ -275,7 +275,9 @@ ENDIF ()
 
 INSTALL (TARGETS usrsctp usrsctp-static DESTINATION ${CMAKE_INSTALL_LIBDIR})
 #INSTALL (DIRECTORY include DESTINATION ${CMAKE_INSTALL_PREFIX})
-INSTALL (FILES ${usrsctp_root_HEADERS} DESTINATION include/usrsctp)
-INSTALL (FILES ${usrsctp_netinet_HEADERS} DESTINATION include/usrsctp/netinet)
-INSTALL (FILES ${usrsctp_netinet6_HEADERS} DESTINATION include/usrsctp/netinet6)
-INSTALL (FILES ${CMAKE_CURRENT_BINARY_DIR}/usrsctp.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+#INSTALL (FILES ${usrsctp_root_HEADERS} DESTINATION include/usrsctp)
+#INSTALL (FILES ${usrsctp_netinet_HEADERS} DESTINATION include/usrsctp/netinet)
```

Comment:
This is exactly what we achieve by this patch.
The behavior before this patch is to copy all files to <includedir>/usrsctp/

With this patch *only* the usrsctp.h is copied  -  and not to <includedir>/usrsctp/ but to <includedir>/ - this is how the traditional method does.